src/vs/platform/environment/common/environment.ts

```diff
@@ -13,6 +13,7 @@ export interface ParsedArgs {
 	_urls?: string[];
 	help?: boolean;
 	version?: boolean;
+	telemetry?: boolean;
```

Comment:
@lramos15 shouldn't break anything but missed this arg when removing the command